mobile/lib/features/welcome/welcome_screen.dart

```diff
@@ -136,21 +129,17 @@ class _WelcomeScreenState extends State<WelcomeScreen> {
                                           color: tenTenOnePurple.shade300.withOpacity(0.2))),
                                   filled: true,
                                   fillColor: tenTenOnePurple.shade300.withOpacity(0.2),
-                                  labelText: 'Email (optional)',
+                                  labelText: 'Nostr Pubkey, Telegram or Email (optional)',
                                   labelStyle: const TextStyle(color: Colors.black87, fontSize: 14),
                                   hintText: 'Let us know how to reach you'),
                               validator: (value) {
                                 if (value == null || value.isEmpty) {
                                   return null;
                                 }
-
-                                if (!isEmailValid(value)) {
-                                  return 'Please enter a valid email address';
-                                }
```

Comment:
Should we at least validate that it is not longer than a certain number of characters? I'm not too keen on letting users insert arbitrary stuff in the database.